lab-brian/src/component/todo-list/index.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import './_todo-list.scss';
+import React from 'react';
+import TodoForm from '../todo-form';
+
+class TodoList extends React.Component {
+  render() {
+    return (
+      <section className='todo-list'>
+        <p className='title'>notes. </p>
+        <ul>
+          {this.props.todos.map((item, i) => {
+            return <li key={i}>
```

Comment:
From a react point of view it is better to add a non-index as the key. React uses this to keep track of changes. My understanding is that for example if you were to remove a note in the middle of your collection it would have to re-render all of the notes that follow it, since their key's would change. But if you used an id like uuid as a prop for the key then it would recognize that those haven't changed, and won't have to re-render. Check me on this, look it up, but that's my understanding.